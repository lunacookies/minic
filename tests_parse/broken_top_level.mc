abcdef
12345
func
==
