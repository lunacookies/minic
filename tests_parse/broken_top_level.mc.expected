func <missing>
	<missing>
tests_parse:0..6: error: expected function but found identifier
tests_parse:7..12: error: expected function but found number literal
tests_parse:18..20: error: expected identifier but found “==”
tests_parse:20..21: error: missing statement
